software/moltres/wiki/input_example.md

```diff
@@ -233,7 +233,7 @@ dimension of the mesh, the argument following `-o` is the name of the output
 Gmsh or Cubit can be used for generation of highly complex mesh
 structures. However, if the user just wishes to generate a simple mesh, he/she
 may use MOOSE's built-in type `GeneratedMesh`. For using `GeneratedMesh` we
-refer the reader [here](http://mooseframework.org/wiki/MooseSystems/Mesh/).
+refer the reader [here](https://mooseframework.inl.gov/source/mesh/GeneratedMesh.html).
```

Comment:
I believe this is the right page, but I am curious to know if there a difference in build between mooseframework.org and mooseframework.inl.gov. @robfairh  @smpark7 thoughts?

Reply:
also @katyhuff thoughts?

Reply:
mooseframework.org is a vanity link for pages hosted by the mooseframework.inl.gov server..